source/fundamentals/data-formats/document-data-format-record.txt

```diff
@@ -191,3 +190,38 @@ as shown in the following code:
 
       NetworkDeviceRecord[deviceId=fedc..., name=Enterprise Wi-fi, deviceType=router]
 
+Operations with Recursive Types
+-------------------------------
+
+The driver natively supports encoding and decoding cyclical
+references without causing runtime recursion. The following code
+provides an example of a recursive class design:
```

Comment:
Issue: My Java's a little rusty, but isn't this a method named RecordTree that returns a record? If not, please ignore :)

Suggestion: Change code sample to a recursive class.

Reply:
This feature is about records, not classes, so I think it should stay as is.